instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/NetServerAttributesGetter.java

```diff
@@ -24,5 +24,5 @@
   Integer peerPort(REQUEST request);
 
   @Nullable
-  String peerIp(REQUEST request);
+  String peerName(REQUEST request);
```

Comment:
@lmolkova do u think this is right for net server attributes (just to use `net.peer.name` and `net.peer.port`)? we used to use `net.peer.ip` because we thought that was better when we knew we had an IP address, but with new proposal my understanding is to put IP address in `net.peer.name` when IP address is all that we have available

Reply:
I probably should clarify it in the spec- `net.peer.name` only applies to client instrumentations. 
Server instrumentations would only set `net.sock.peer.addr`, `net.sock.peer.port` to the direct peer address (not x-forwarded-for or similar)